requirements.yml

```diff
@@ -31,7 +31,7 @@
   version: 2.0.5
 
 - src: openmicroscopy.omero-web
-  version: 1.0.3
+  version: 2.0.0
```

Comment:
This is already the version in master

Reply:
@pwalczysko included it in his PR 
https://github.com/openmicroscopy/prod-playbooks/commit/66a6a6f7adec7eb2f341e6ce3fb88f29323a3609

Reply:
@manics 2.0.1 is now available if you want to bump it in this PR